Mobile/iOS/Templates/xxx-share_edit_privileges.md

```diff
@@ -0,0 +1,49 @@
+13|Can delete folder|1. Swipe to share a folder<br>2. Select a user/group and grant it to delete<br>3. Login with the sharee<br>4. Try to delete a file into the folder| 3. The sharee can see the file<br>4. The file is correctly deleted
+14|Can not delete folder|1. Swipe to share a folder<br>2. Select a user/group but does not grant it to delete<br>3. Login with the sharee<br>4. Try to delete a file into the folder| 3. The sharee can see the file<br>4. The file can not be deleted
+15|Can not edit folder|1. Swipe to share a folder<br>2. Select a user/group but does not grant it to edit<br>3. Login with the sharee<br>4. Try to edit files (create/change/delete) with an external app| 3. The sharee can see the file<br>4. The file can not be edited
+16|Stop sharing|1. Swipe to share a file/folder<br>2. Select a user/group and grant it some privileges<br>3. Login with the sharee<br>4. Login again with the sharing user and long press to handle sharing privileges<br>5. Tap on "stop sharing"| 3. The sharee can see the file<br>5. Check that the sharee can not see the file/folder
+17|Privileges in server|1. Login on server<br>2. Grant a file/folder some privileges<br>3. Login on app| Check that the granted privileges on server are visible in mobile app
+18|Several sharees|1. Share a file/folder with several users granting them different privileges<br>|Each sharee can only manage the file/folder according his privileges
+19|Several files|1. Share several files/folders with a user/group granting him different privileges<br>|The user/group can only manage the files/folders according his privileges
+**Errors**|
+20|No connection|1. Swipe to share a file/folder with privileges<br>2. Switch the device connection off<br>3. Change the privileges<br>4. Tap on "done"| An error is shown
+21|Server down|1. Swipe to share a file/folder with privileges<br>2. Server down<br>3. Change the privileges<br>4. Tap on "done"| An error is shown
+22|Sharing API disabled|1. Swipe to share a file/folder with privileges<br>2. In server, disable the sharing API<br>3. Change the privileges<br>4. Tap on "done"| An error is shown
+23|Resharing disabled|1. Swipe to share a file/folder with privilege of sharing<br>2. In server, disable the resharing<br>3. Login with the sharee<br>4. Try to share the file/folder by handling privileges<br>5. Tap on "done"| An error is shown
+24|Changing password|1. Swipe to share a file/folder with privileges<br>2. In server, change the password of the user<br>3. Try to share the file/folder by handling privileges<br>4. Tap on "done"| Credentials are required
+25|Delete file|1. Swipe to share a file/folder with privileges<br>2. In server, delete the file<br>3. Tap on "done"|An error is shown
```

Comment:
This test case should have been tested within the internal sharing, I mean, I don't see the point of checking now, since it´s not related to privileges which is the main aim of this feature, just an idea same on the next test case 26